src/pages/MainPage.vue

```diff
@@ -36,7 +36,7 @@
         :roads="roads"
         :active-road="activeRoad"
         :subjects="subjectsInfo"
-        @delete-road="$refs.authcomponent.deleteRoad($event)"
+        @devare-road="$refs.authcomponent.devareRoad($event)"
```

Comment:
Is this supposed to be renamed?

Reply:
Nope, result of bad find+replace. Thanks for catching it!